engines/libpmem.c

```diff
@@ -193,6 +193,7 @@ static enum fio_q_status fio_libpmem_queue(struct thread_data *td,
 
 	dprint(FD_IO, "DEBUG fio_libpmem_queue\n");
 	dprint(FD_IO,"td->o.odirect %d td->o.sync_io %d \n",td->o.odirect, td->o.sync_io);
+	/* probably fine to treat both O_SYNC / DSYNC as NODRAIN */
 	flags = td->o.sync_io ? 0 : PMEM_F_MEM_NODRAIN;
```

Comment:
behavior seems fine, but the comment is the other way around ;) if `td->o.sync_io` is set (to either one) than we don't want to set up the flag "NODRAIN"

Reply:
Oops, you're right of course. Should be fixed now.